### DISCUSSION TITLE
Integrating parler-tts

### DISCUSSION THREAD

**Discussion by CeciliaXCIX** [LostRuins/koboldcpp#783] [Ideas]
Would it be possible to integrate [parler-tts](https://github.com/huggingface/parler-tts)?

**Comment by LostRuins** [comment]
Wow, that model does look impressive. However, right now only a pytorch implementation is available, which can't really be used directly from kobold. Either it will have to be ported to GGML, or a separate API server would need to be created with this capabilities wrapped around it. 

I'll keep an eye out, however right now your current options will be the existing TTS within Kobold Lite, which are the inbuilt browser TTS, XTTS API server, and AllTalk.
